### PR TITLE
feat(mknod): add completion

### PR DESCRIPTION
**Pull request by mamercad**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

New completion

**What is the current behavior? (You can also link to an open issue here)**

There is no completion for `mknod`

**What is the new behavior (if this is a feature change)?**

There will be completion for `mknod`

**Additional info:**

`man mknod`

### CODE REVIEW

File: src/mknod.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    {
+      name: "c",
+      description: "Create (c)haracter device",
+      exclusiveOn: ["b"],
+    },
+    {
+      name: "b",
+      description: "Create (b)lock device",
+      exclusiveOn: ["c"],
+    },
```
Comment (fedeci) [inline src/mknod.ts:44]: These are subcommands since they are mutually exclusive and do not have a dash.

File: src/mknod.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    {
+      name: "major minor",
+      description:
+        "The major device number is an integer number which tells the kernel which device driver entry point to use; the minor device number tells the kernel which one of several similar devices the node corresponds to; for example, it may be a specific serial port or pty",
+    },
+    {
+      name: "major unit subunit",
+      description:
+        "The major device number is an integer number which tells the kernel which device driver entry point to use; the unit and subunit numbers select a subset of a device; for example, the unit may specify a particular SCSI disk, and the subunit a partition on that disk",
+    },
```
Comment (fedeci) [inline src/mknod.ts:60]: These appear to be the same arg, likely the structure you want is: two required args (major and minor) and one optional arg (subunit)

**Review by fedeci — CHANGES_REQUESTED** [review]
Thanks! Can you resolve the comments please?

File: src/mknod.ts (hunk 2 above)
Comment (mamercad) [reply src/mknod.ts:60]: Hrm, I'm not sure that's right, the second is "major unit subunit" (not "major minor").

Comment (mamercad) [reply src/mknod.ts]: I've pushed an idea...

Comment (fedeci) [reply src/mknod.ts]: Looks better!

**Review by fedeci — APPROVED** [review]

**Pull request merged**